lib/polarshift/request.rb

```diff
@@ -307,6 +305,22 @@ def update_caseruns(project_id, run_id, updates)
         )
       end
 
+      def update_caseruns_smart(project_id, run_id, updates, timeout: 360)
+        res = nil
+        success = wait_for(timeout, interval: 30) {
+          res = update_caseruns(project_id, run_id, updates)
+          if res[:success]
+            return res
+          elsif res[:error]
```

Comment:
I think a straight `else` should work here. Do you have any specific reason to use `elsif res[:error]`?